shinydashboard/lantern/global.R

```diff
@@ -35,7 +37,7 @@ ui_special_values <- list(
 # The list of fhir versions and vendors are unlikely to change during a user's session
 # we'll update them on timer, but not refresh the UI
 app <<- list(
-  fhir_version_list      = get_fhir_version_list(endpoint_export_tbl),
+  fhir_version_list      = reactiveVal(get_fhir_version_list(endpoint_export_tbl)),
```

Comment:
I was wondering this as well. I actually made the mistake of using the app_data fhir_version_list instead of the app fhir_version_list and was getting null errors until I released the one being used throughout the application is the app fhir_version_list. Therefore, I am not sure if the one in the app_data was maybe previously used but then instead was moved to app? I can delete the one from app_data, but I am not sure why it is defined in both of them so I am not sure if that would cause any problems

Reply:
I think this might just have been a copy/paste error when moving values into the app and app_data lists. app$fhir_version_list is what was supposed to be used.

Reply:
Thanks Mike! I will remove the app_data$fhir_version_list!